src/Octoshift/GithubApi.cs

```diff
@@ -105,6 +105,20 @@ public virtual async Task RemoveTeamMember(string org, string teamSlug, string m
             await _retryPolicy.HttpRetry(() => _client.DeleteAsync(url), _ => true);
         }
 
+        public virtual async Task<bool> DoesRepoExist(string org, string repo)
```

Comment:
I would also like to see a unit tests for this.

Reply:
Addressed in [`eb2601b`](https://github.com/github/gh-gei/pull/787/commits/eb2601b00ae9bb68e82fe03f158415332a9200e0)